### DISCUSSION TITLE
How to improve the the Dp model with low RMSE?

### DISCUSSION THREAD

**Discussion by Guancred** [deepmodeling/deepmd-kit#3722] [Q&A]
I have built the dp model by the code of dpgen, then collecting the data for a long training (7000000), while the energy and force RMSE are 60 meV/atom and 350 meV/A.

So, if i want to improve this dp model and decrease the RMSE, which training parameters should be adjusted ?

Thank you very much!

**Comment by wanghan-iapcm** [comment]
you may try with dpa-1 (with attention layers) https://github.com/deepmodeling/deepmd-kit/blob/r2/examples/water/se_atten/input.json
and dpa-2 https://github.com/deepmodeling/deepmd-kit/blob/devel/examples/water/dpa2/input_torch.json

**Reply by Guancred** [reply]
Thank you, i have tried with the dpa-1, while the RMSE still show a large value, 
![image](https://github.com/deepmodeling/deepmd-kit/assets/91767936/18f6868c-26d6-49bd-a3bb-c3a4889b2322)

and the input.json:
{
  "model": {
    "descriptor": {
      "type": "se_atten",
      "sel": 140,
      "rcut_smth": 0.5,
      "rcut": 7,
      "neuron": [
        25,
        50,
        100
      ],
      "resnet_dt": false,
      "axis_neuron": 16,
      "attn":128,
      "attn_layer":2,
      "attn_mask":false,
      "attn_dotr":true,
      "type_one_side": true,
      "seed": 1541804686,
      "_activation_function": "tanh"
    },
    "fitting_net": {
      "neuron": [
        240,
        240,
        240
      ],
      "resnet_dt": true,
      "seed": 1477278577

Any parameters should be revised for the decreased RMSE? Thanks very much!

**Comment by wanghan-iapcm** [comment]
apart from the mentioned model architectures, you may follow the doc https://docs.deepmodeling.com/projects/deepmd/en/r2/troubleshooting/precision.html#why-does-a-model-have-low-precision to improve the accuracy of the model

**Reply by Guancred** [reply]
Hi, whether the "compress" is not support for the DPA-1 model?  when i compress it, an error will be occur: "can not compress model when attention layer is not 0."

**Reply by wanghan-iapcm** [reply]
only compressible when `attn_layer` is set to 0

**Reply by Guancred** [reply]
Thanks, so, i should set the "attn_layer: 0", and then train another model if i want to compress?
